.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **Attention:** if this is a _question_ about how to build / test / query / deploy using
+        Bazel intellij, or a _discussion starter_, send it to bazel-discuss@googlegroups.com or discuss on
```

Comment:
I think we should direct the users to this group instead: [intellij-bazel-plugin@googlegroups.com](mailto:intellij-bazel-plugin@googlegroups.com)